projects/cppcheck/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER daniel.marjamaki@gmail.com
+
+RUN git clone https://github.com/orbitcowboy/afl_cppcheck.git afl_cppcheck
```

Comment:
nit: is there any plan to move code from https://github.com/orbitcowboy/afl_cppcheck.git into the main dev repo https://github.com/danmar/cppcheck.git somepoint in future. is @orbitcowboy a cppcheck dev ?

Reply:
I do not have such a plan. Yes, orbitcowboy is an active cppcheck dev.

Reply:
No need then since orbitcowboy@ is active dev.